### PR TITLE
Standardize '/version' endpoint on all services

### PR DESCRIPTION
**Pull request by vgrassia**
## Type of change
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

This PR adds a `/version` endpoint to all of our services to easily check what version is running.  This is very helpful after deploying a new version to confirm the new version is running.

## Code changes

* **bitwarden_license/src/Sso/Controllers/HomeController.cs:** Removed `/alive` and `/now` endpoints.
* **bitwarden_license/src/Sso/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `HomeController`.

* **src/Admin/Controllers/HomeController.cs:** Removed `/alive` and `/now` endpoints.
* **src/Admin/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `HomeController`.

* **src/Api/Controllers/MiscController.cs:** Removed `/alive`, `/now`, and old `/version` endpoints.
* **src/Api/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `MiscController`.
* **src/Core/Models/Api/Response/VersionResponseModel.cs:** Deleted as it is no longer used by the `API` service.

* **src/Billing/Controllers/HomeController.cs:** Deleted as it is no longer needed.
* **src/Billing/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `HomeController`.

* **src/Events/Controllers/CollectController.cs:** Removed `/alive` and `/now` endpoints.
* **src/Events/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `CollectController`.

* **src/Icons/Controllers/IconsController.cs:** Removed `/alive` and `/now` endpoints.
* **src/Icons/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `IconsController`.

* **src/Identity/Controllers/MiscController.cs:** Deleted as it is no longer needed.
* **src/Identity/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `MiscController`.

* **src/Notifications/Controllers/SendController.cs:** Removed `/alive` and `/now` endpoints.
* **src/Notifications/Controllers/InfoController.cs:** Added `InfoController` with `/version` endpoint that returns a JSON string of the current service version. Moved `/alive` and `/now` endpoints from `SendController`.

### CODE REVIEW

**Review by cscharf — CHANGES_REQUESTED** [review]
Hi @vgrassia, thank you for tackling this! Just a few things:

* Please see the standard means for collecting the current version, this is already in a [Core helper](https://github.com/bitwarden/server/blob/d3673cdc8513a0846694ae0aff6b205f42838c73/src/Core/Utilities/CoreHelpers.cs#L563)
* Please add the same endpoint to the HomeController of the Admin site/project/service (and normalize the `GetAlive()` endpoint
* On the note of normalization, it would be a beautiful thing to get the Alive, Now and Version endpoints in a consistently named controller... Perhaps a new controller in each project named `HealthController` or `InfoController` and moving/adding into there those endpoints so it's absolutely consistent across all projects.

File: src/Api/Controllers/MiscController.cs [hunk 1]
```diff
@@ -24,19 +23,6 @@ public class MiscController : Controller
...
-        }
-
-        [HttpGet("~/version")]
-        public VersionResponseModel Version()
-        {
-            return new VersionResponseModel();
-        }
-
         [HttpGet("~/ip")]
         public JsonResult Ip()
```
Comment (cscharf) [inline src/Api/Controllers/MiscController.cs:27]: Maybe move the `Ip()` method as well?

**Review by cscharf — COMMENTED** [review]
okay, one suggestion and then there's 1 service missing (EventsProcessor), see: https://github.com/bitwarden/server/blob/master/src/EventsProcessor/Startup.cs#L51

File: src/Api/Controllers/MiscController.cs (hunk 1 above)
Comment (vgrassia) [reply src/Api/Controllers/MiscController.cs]: Done.

**Review by cscharf — APPROVED** [review]
awesome!

**Pull request merged**